### PR TITLE
Play/Pause problem on lists fixed. Bind-X-Row methods reduced to one, adding type parameter.

### PR DESCRIPTION
**Pull request by srcnalt**
See commit desc.

### CODE REVIEW

File: assets/js/app/collection.js [hunk 1]
```diff
@@ -180,68 +182,51 @@ var songCollection = {
...
+				aurousScript.player.changeMedia(title, artist, albumArt, url);
+				aurousScript("#playerPause").show();
+				aurousScript("#playerPlay").hide();
+				window.previousCollectionRow = parent;
+				window.previousCollectId = id;
+			}
+			else{	//checking if the same item clicked
+				if(aurousScript.player.paused){
+					window.previousCollectionRow.addClass("result-now-playing");
+					aurousScript("#" + type + "-row-icon-" + id).html("pause");
```
Comment (yasaricli) [inline assets/js/app/collection.js]: why window ?

File: assets/js/app/collection.js [hunk 2]
```diff
@@ -180,68 +182,51 @@ var songCollection = {
...
-            aurousScript.player.changeMedia(title, artist, albumArt, url);
-            aurousScript("#playerPause").show();
-            aurousScript("#playerPlay").hide();
-            window.previousCollectionRow = parent;
-            window.previousCollectId = id;
-
-        });
+            var id = parent.attr(type + '-data-id');
+			
+			if(id != window.previousCollectId){
```
Comment (yasaricli) [inline assets/js/app/collection.js]: use 

    !utils.isEqual(id, previousCollectId)

File: assets/js/app/collection.js [hunk 3]
```diff
@@ -180,68 +182,51 @@ var songCollection = {
...
+				window.previousCollectionRow = parent;
+				window.previousCollectId = id;
+			}
+			else{	//checking if the same item clicked
+				if(aurousScript.player.paused){
+					window.previousCollectionRow.addClass("result-now-playing");
+					aurousScript("#" + type + "-row-icon-" + id).html("pause");
+					window.currentCollectionId = id;
+		
+					aurousScript("#playerPause").show();
```
Comment (yasaricli) [inline assets/js/app/collection.js]: use 

    aurousScript.showControl('Pause');

File: assets/js/app/collection.js [hunk 4]
```diff
@@ -180,68 +182,51 @@ var songCollection = {
...
+					aurousScript("#playerPlay").hide();
+				}
+				else{
+					window.previousCollectionRow.removeClass("result-now-playing");
+					aurousScript("#" + type + "-row-icon-" + id).html("play_arrow");
+							
+					aurousScript("#playerPause").hide();
```
Comment (yasaricli) [inline assets/js/app/collection.js]: use 

    aurousScript.showControl('Play');

File: assets/js/app/collection.js [hunk 5]
```diff
@@ -180,68 +182,51 @@ var songCollection = {
...
+					window.previousCollectionRow.removeClass("result-now-playing");
+					aurousScript("#" + type + "-row-icon-" + window.previousCollectId).html("play_arrow");
+				}
+				
+				window.currentCollectionId = id;
+				aurousScript("#" + type + "-row-icon-" + id).html("pause");
+
+				var url = parent.attr(type + '-data-value');
+				aurousScript.player.changeMedia(title, artist, albumArt, url);
+				aurousScript("#playerPause").show();
```
Comment (yasaricli) [inline assets/js/app/collection.js]: use 

   aurousScript.showControl('Pause');

**Comment by srcnalt** [comment]
@yasaricli 
- why *window*
check the method I changed, you'll se why.

- use *!utils.isEqual...*
```
try{
	utils.isEqual(1, 2);
}catch(e){
	alert(e.message);
}
```
*blink!* undefined is not a function

- use *aurousScript.showControl...*
```
try{
	aurousScript.showControl('Pause');
}catch(e){
	alert(e.message);
}
```
*blink!* undefined is not a function
Also I have no problem with using global vars.

Well, mate...

**Pull request merged**